app.js

```diff
@@ -33,12 +33,20 @@ function move({
     });
 }
 
-function deletePath({ path }) {
+async function deletePath({
+  path,
+  securely,
+}) {
+  const pathExists = await fs.pathExists(path);
+  if (pathExists && securely) {
```

Comment:
If doesn't exist then remove?

Reply:
If it doesn't exist then there's no need to perform that operation. We shouldn't base our logic on the underlying framework/library.
Ask Kenny what should be the behavior in such situation - ignore, message or error, and add it in such case.